.golangci.yaml

```diff
@@ -1,12 +1,10 @@
 linters-settings:
   depguard:
-    list-type: denylist
-    packages:
-      # logging is allowed only by logutils.Log, logrus
-      # is allowed to use only in logutils package
-      - github.com/sirupsen/logrus
-    packages-with-error-message:
-      - github.com/sirupsen/logrus: "logging is allowed only by logutils.Log"
+    rules:
+      logger:
+        deny:
+          - pkg: github.com/sirupsen/logrus
+            desc: use github.com/NethermindEth/juno/utils for logging
```

Comment:
What is `depguard` used for? And how does this change fix it?

Reply:
`depguard` is used to restrict what dependencies are added (e.g., disallow alternative logging packages).

`depguard` was upgraded in `golangci-lint`, which changed the expected configuration layout. This new layout is aligned with `depguard/v2`.